src/index.ts

```diff
@@ -298,41 +110,24 @@ export default class OtaClient {
     /**
      * Returns file translations
      *
-     * @param file file from distribution
+     * @param file file content path
      * @param languageCode language code
      */
-    async getFileTranslations(file: string, languageCode?: string): Promise<string | any | null> {
-        let url = `${OtaClient.BASE_URL}/${this.distributionHash}/content`;
-        const language = this.getLanguageCode(languageCode);
-        const [languageMappings, customLanguages, apiLanguages] = await Promise.all([
-            this.getLanguageMappings(),
-            this.getCustomLanguages(),
-            this.getLanguages(),
-        ]);
-        const languageMapping = (languageMappings ?? {})[language];
-        const customLanguage = (customLanguages ?? {})[language];
-        if (includesLanguagePlaceholders(file)) {
-            url += replaceLanguagePlaceholders(file, language, apiLanguages, languageMapping, customLanguage);
-        } else {
-            url += `/${language}${file}`;
-        }
+    async getFileTranslations(file: string): Promise<string | any | null> {
```

Comment:
I'm afraid this method might be confusing since users probably will try passing the file path without the `content/` prefix. Do you have any ideas on how to avoid this confusion? Some validation?

Reply:
In docs it’s stated that file path that is returned from ‘getContent’ should be used here. Also I can add validation to check if given file is present in files paths in content. And if not then throw an error with meaningful message

Reply:
an error would be great